package.json

```diff
@@ -54,9 +54,11 @@
     "@babel/eslint-parser": "^7.24.7",
     "eslint": "8.56.0",
     "jest": "^29.7.0",
+    "jest-environment-node-single-context": "^29.4.0",
```

Comment:
What does this do?

Reply:
This mitigates the error that appears when running `yarn test`. The `fs-temp`'s utility chain of imports leads to a package named `to-data-view` which performs the following check:
<img width="804" alt="image" src="https://github.com/user-attachments/assets/098dd04d-aafd-4822-9e83-034c159d2219">

Somehow, the data gets transformed into a `Uint8Array` and gets passed to this function, where it yields false for these checks ONLY in `jest`. Spending some time on Github issues, I've found a suggestion that this is caused by `jest` and `node` having different contexts during the test suite run. This testing environment dependency helped to solve this issue.
I've tried testing the same test with the same function that uses `fs-temp` to create temporary files in a testing project of my own and it was not happening, but in `jscodeshift` codebase it somehow did. So, instead of trying to update `jest` and/or write a custom environment file, I used this existing package.

Reply:
You know what, I removed the `fs-temp` dependency too. Let me know if the new implementation needs improvements.